cmd/cli/app/repo/repo_status.go

```diff
@@ -0,0 +1,122 @@
+//
+// Copyright 2023 Stacklok, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package repo
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"google.golang.org/grpc"
+
+	"github.com/stacklok/minder/cmd/cli/app"
+	"github.com/stacklok/minder/internal/util"
+	"github.com/stacklok/minder/internal/util/cli"
+	minderv1 "github.com/stacklok/minder/pkg/api/protobuf/go/minder/v1"
+)
+
+var statusCmd = &cobra.Command{
+	Use:   "status",
+	Short: "Get repository evaluation status",
+	Long:  `The repo status subcommand is used to get the evaluation status for a registered repository within Minder.`,
+	RunE:  cli.GRPCClientWrapRunE(statusCommand),
+}
+
+// statusCommand is the repo evaluation status subcommand
+func statusCommand(ctx context.Context, cmd *cobra.Command, conn *grpc.ClientConn) error {
```

Comment:
My $0.02 would be to avoid implementing the CLI command until we have a working backend, so end-users don't pick it up and try to use it by accident.  In the meantime, `grpcurl` can be used to send requests.  But I'm open to other ideas.

Reply:
Good point 👍 I'll hold that off for once we complete the implementation